test-libs/good/.gitignore

```diff
@@ -0,0 +1,18 @@
+# Windows Files
+Desktop.ini
+
+# Mac Files
+.DS_Store
+
+# Compiled files
+*.o
+*.so
+*.rlib
+*.dll
```

Comment:
`target` isn't ignored right? That's because you have `/target/` in the `.gitignore`.